api/qrcode.php

```diff
@@ -2,31 +2,32 @@
 require_once '../lib/config.php';
 require_once '../lib/helper.php';
 
-$filename = $_GET['filename'];
+$filename = $_GET['share_link'];
 
 if ($config['qr']['append_filename']) {
-    $url = $config['qr']['url'] . $filename;
+	$url = $filename;
 } else {
-    $url = $config['qr']['url'];
+	$url = $config['qr']['url'] . "/s/qyGko4KNoZsJY39";
```

Comment:
What's the `/s/qyGko4KNoZsJY39` for?

Reply:
Apologies that was missed from early testing. It was just a fall back share url from when this was just for me. It all has been moved to sharlink.php